packages/api/tests/endpoints/workflows.js

```diff
@@ -73,3 +107,70 @@ test('CUMULUS-912 GET with pathParameters and with an unauthorized user returns
     assertions.isUnauthorizedUserResponse(t, response);
   });
 });
+
+test('GET with no path parameters and an authorized user returns a list of workflows', async (t) => {
+  const request = {
+    httpMethod: 'GET',
+    headers: authHeaders
+  };
+
+  return testEndpoint(workflowsEndpoint, request, (response) => {
+    t.is(response.statusCode, 200);
+    const results = JSON.parse(response.body);
+
+    t.deepEqual(results, workflowList);
+  });
+});
+
+test('GET an existing workflow with an authorized user returns a specific workflow', async (t) => {
+  const request = {
+    httpMethod: 'GET',
+    pathParameters: {
+      name: 'HelloWorldWorkflow'
+    },
+    headers: authHeaders
+  };
+
+  return testEndpoint(workflowsEndpoint, request, (response) => {
+    t.is(response.statusCode, 200);
+    const result = JSON.parse(response.body);
```

Comment:
There should be a blank line separating the test on line 135 from the unrelated statement on line 136.  It would be better grouped like

```javascript
t.is(response.statusCode, 200);

const result = JSON.parse(response.body);
t.deepEqual(result, workflowList[0]);
```

Reply:
changed